src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+
+    }
+
+    @Override
+    public void stopWork() {
+
```

Comment:
The `stopWork` method should print a message indicating that the Excavator has stopped working. Please add a print statement here.